test/support.py

```diff
@@ -50,9 +50,23 @@ class TestFoo(HPyTest):
         hpy_abi = make_hpy_abi_fixture('with hybrid', class_fixture=True)
     """
     if ABIs == 'default':
-        ABIs = ['cpython', 'universal', 'debug']
+        try:
+            if sys.implementation.name == 'cpython':
+                ABIs = ['cpython', 'universal', 'debug']
+            else:
+                ABIs = ['universal', 'debug']
+        except:
+            # CPython2.7: no sys.implementation
+            ABIs = ['cpython', 'hybrid', 'hybrid+debug']
```

Comment:
```suggestion
            ABIs = ['cpython', 'universal', 'debug']
```
The list here was copy and pasted incorrectly.

We should simplify this somehow. Perhaps there can just be a pytest flag that adds `cpython` to the list? Or make a flag here like `is_cpython = not hasattr(sys, "implementation") or sys.implementation.name == 'cpython'` so at least we're not repeating strange logic.